helm/cluster-openstack/files/etc/patches/kube-apiserver-patch.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+#
+# Creates kube-apiserver+json.json file by replacing
+# environment variables in kube-apiserver+json.tpl
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+if [ "$#" -ne 1 ]; then
+    echo "Illegal number of parameters" >&2
+    echo "Usage: bash kube-apiserver-patch.sh <resource-ratio>" >&2
+    exit 1
+fi
+
+ratio=$1
+
+cpus="$(grep -c ^processor /proc/cpuinfo)"
```

Comment:
But this values are taken always from MC? or it will work for WCs too?

Reply:
It will run for both MC and WC. In the control-plane nodes of the cluster.